homework/05LessonFive.js

```diff
@@ -63,15 +63,15 @@ for (forLoopCounter = 0; forLoopCounter > -10; forLoopCounter--) {
 
 // Problem 8:
 // Fill in forLoopArrayResult so that it matches forLoopArray
-var forLoopArrayResult = [];
+var forLoopArrayResult = [-10,-9,-8,-7,-6,-5,-4,-3,-2,-1];
 
 // ****
 // Concept Checkpoint
 //
 // Write your answer in comments
 //
 // What are loops? Why do we use them?
-//
+// Repeats until a certain condition is met.
 
 // ****
 // Tests
```

Comment:
You haven't pushed up apps 05 & 06???

![what](https://year13.com.au/wp-content/uploads/2016/01/dafuq.jpg)